src/signalcorr.jl

```diff
@@ -77,6 +83,14 @@ function autocov!{T<:RealFP}(r::RealMatrix, x::Matrix{T}, lags::IntegerVector; d
     return r
 end
 
+
+"""
+    autocov(x, [lags]; demean=true)
+
+Compute the autocovariance of a vector or matrix `x`, optionally specifying
```

Comment:
Should say a word about the default `lags`. I see it's computed as `0 : min(size(x,1)-1, round(Int,10*log10(size(x,1))))`, but I really don't know what's the logic behind this. Looks similar to R, though, and their docs don't explain why either. So maybe just copy that formula in a second paragraph (without the `round` to make it more readable?).